app/src/main/res/values/strings.xml

```diff
@@ -21,5 +21,6 @@
     <string name="dialog_message">If you exit now, the changes will be lost</string>
     <string name="dialog_negative_button">Discard changes</string>
     <string name="dialog_positive_button">Continue editing</string>
+    <string name="authority">com.example.criminalintentapp.fileprovider</string>
```

Comment:
I saw you tried to use this when creating the `photoUri`. `strings.xml` is not the right place for strings like this, usually this file is localized, meaning that these strings will get translated to all the languages the app supports. There's a `translatable="false"` flag you can set to avoid that, but still, strings like `authority`(not displayed in the UI) can be extracted to `private const val`s where they're used